lib/bolt/outputter/human.rb

```diff
@@ -174,11 +174,22 @@ def print_result(result)
           if result.message?
             @stream.puts(remove_trail(indent(2, result.message)))
           end
-
           case result.action
           when 'command', 'script'
             safe_value = result.safe_value
-            @stream.puts(indent(2, safe_value['merged_output'])) unless safe_value['merged_output'].strip.empty?
+            if safe_value["merged_output"]
+              @stream.puts(indent(2, safe_value['merged_output'])) unless safe_value['merged_output'].strip.empty?
+
+            else # output stdout or stderr
+              unless safe_value['stdout'].strip.empty?
```

Comment:
```suggestion
              unless  safe_value['stdout']&.strip.empty?
```
This is the [Ruby safe navigation operator](https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) which will just ensure that calling `strip()` doesn't raise an error if `safe_value['stdout']` returns nil.

Reply:
Got it!

Reply:
I think we actually want this to be `unless safe_value['stdout'].nil? || safe_value['stdout'].strip.empty?`, otherwise a `nil` value will still print `STDOUT:`. Same for `stderr`.

Reply:
That makes sense. Just put it in updated commit